semver_tests/src/test_cases/enum_repr_int_removed.rs

```diff
@@ -49,3 +49,47 @@ pub enum UsizeEnum {
     Bar,
     Baz,
 }
+
+#[cfg(not(feature = "enum_repr_int_removed"))]
+#[repr(u8, C)]
+pub enum U8CEnum {
+    Bar,
+    Baz,
+}
+
+#[cfg(feature = "enum_repr_int_removed")]
+#[repr(C)]
+pub enum U8CEnum {
+    Bar,
+    Baz,
+}
+
+#[cfg(not(feature = "enum_repr_int_removed"))]
+#[repr(u8)]
+#[repr(C)]
+pub enum SeparateU8CEnum {
+    Bar,
+    Baz,
+}
+
+#[cfg(feature = "enum_repr_int_removed")]
+#[repr(C)]
+pub enum SeparateU8CEnum {
+    Bar,
+    Baz,
+}
+
+#[cfg(not(feature = "enum_repr_int_removed"))]
+#[repr(u8)]
+#[repr(C)]
+pub enum SeparateU8ToU8CEnum {
+    Bar,
+    Baz,
+}
+
+#[cfg(feature = "enum_repr_int_removed")]
+#[repr(u8, C)]
+pub enum SeparateU8ToU8CEnum {
+    Bar,
+    Baz,
+}
```

Comment:
```suggestion
}

```
It's nice to end files in a newline, since that makes it more likely that there won't be any interoperation issues with other programs (more info here: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline ).

It's very likely that your editor can be configured to do this automatically when you save the file.